props/prop_base.h

```diff
@@ -1376,6 +1376,31 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
       }
       return true;
     }
+    if (!strcmp(cmd, "swing")) {
+      SaberBase::DoEffect(EFFECT_ACCENT_SWING, 0);
```

Comment:
or... would it be better to just do `prop.Event(BUTTON_NONE, EVENT_SWING);` here, and below, and then SaberBase will handle what EFFECT to do, then SB_Effect will pick it up?
I am not sure that is how it would happen.

Reply:
It's an interesting thought.
I'm going to have to think about that for a bit, because I'm not sure which is better.
A lot of the props already have some sort of swing detection in them for "swing on" and similar things, so I guess the question is if we can (and should) combine some of this stuff so that swings move from motion->prop->prop base->sound/blade in a similar way that buttons and clashes do.

Reply:
Well the way it is at the moment, having both lines behaves as intended.
If OFF, "swing" will do swingon. if on, does EFFECT_ACCENT_SWINGThe "drawback" is it also triggers any EFFECT_ACCENT_SWING in the blade style.